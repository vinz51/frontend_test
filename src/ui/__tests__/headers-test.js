import Header, { H1Header } from "../headers";
import renderer from "react-test-renderer";
import React from "react";

describe("Headers", () => {
  describe("Snapshots", () => {
    it("renders header", () => {
      const myTreeComponent = renderer.create(<Header />).toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });

    it("renders title in header", () => {
      const myTreeComponent = renderer.create(<H1Header />).toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
