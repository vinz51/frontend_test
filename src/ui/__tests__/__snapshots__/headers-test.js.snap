// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers Snapshots renders header 1`] = `
<header
  className="sc-AxhUy duBjVs"
/>
`;

exports[`Headers Snapshots renders title in header 1`] = `
<h1
  className="sc-AxjAm sc-AxgMl btHVmf"
/>
`;
